lib/resolve-deps.js

```diff
@@ -1,76 +1,86 @@
 "use strict";
+import path from 'path';
+import _fs from 'fs';
+import _ from 'lodash';
+import B from 'bluebird';
 
-var path = require('path'),
-    fs = require('fs'),
-    _ = require('lodash'),
-    root = path.resolve(__dirname, "../uiauto");
+let fs = {
+  readFile: B.promisify(_fs.readFile)
+};
+
+let root = path.resolve(__dirname, '../uiauto');
 
 // keep a list of files we examine for the purposes of making sure we're
 // not adding the same file to the collated version twice
-var filesExamined;
+let filesExamined;
 
 // this regex helps us get the file path of an import
-var importRe = /^#import ('|")([^('|")]+)('|")$/mg;
+let importRe = /^#import ('|")([^('|")]+)('|")$/mg;
 
-var getDepsForFile = function (file, extraImports) {
+async function getDepsForFile (file, extraImports) {
   if (_.contains(filesExamined, file)) {
-    throw new Error("Re-examining file " + file + "; you need to make sure " +
-                    "the graph is set up so we don't require files twice");
+    throw new Error(`Re-examining file ${file}; you need to make sure ` +
+                    `the graph is set up so we don't require files twice`);
   }
   filesExamined.push(file);
-  var data = fs.readFileSync(file).toString('utf8');
-  var deps = {};
+  
+  let data = await fs.readFile(file, 'utf8');
+  let deps = {};
   deps[file] = [];
```

Comment:
Yeah, you [can](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-object-initializer) finally have computed keys in object initialization!